llvm_mode/pass/AngoraPass.cc

```diff
@@ -524,9 +524,16 @@ void AngoraLLVMPass::resetIndirectCallContext(IRBuilder<> *IRB) {
 }
 
 void AngoraLLVMPass::processCall(Instruction *Inst) {
-  
-  CallInst *CI = dyn_cast<CallInst>(Inst);
-  Function* fp = CI->getCalledFunction();
+
+  Function* fp;
+  if (isa<CallInst>(Inst)) {
```

Comment:
This check is redundant, `dyn_cast` below returns `nullptr` if the cast fails. Just check for that in the `if`.